recipes/czmq/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "czmq" %}
+{% set version = "4.1.1" %}
+{% set sha256 = "b7623433547aa4b6e79722796c27ebc7c0470fea4204e920fd05e717c648f889" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zeromq/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - libzmq2zeromq.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - pkg-config  # [unix]
+    - libuuid  # [unix]
```

Comment:
libuuid and pcre as well as zeromq should be in a host section, afaik

Reply:
Thank you for catching that. The most recent commit moves those packages and libuuid to the host section.